components/PlantDeleteButton/index.js

```diff
@@ -0,0 +1,77 @@
+import { useState } from "react"
+import styled from "styled-components";
+
+export default function PlantDeleteButton({ name, id, handleDeletePlant }) {
+const [showDeleteModal, setShowDeleteModal] = useState(false);
+
+function handleToggleModal() {
+    setShowDeleteModal(!showDeleteModal);
+};
+
+return (
+    <>
+    <StyledButton type="button" onClick={handleToggleModal}>Delete Plant</StyledButton>
+
+    {showDeleteModal === true ? (
+        <StyledDeleteModalBackground>
+            <StyledDeleteModal>
+                <p>Do you really want to delete <StyledPlantName>{name}</StyledPlantName>?</p>
+                <StyledDeleteModalButtonContainer>
+                    <StyledButton type="button" onClick={handleToggleModal}>Cancel</StyledButton>
+                    <StyledButton type="button" onClick={() => handleDeletePlant(id)}>Delete</StyledButton>
+                </StyledDeleteModalButtonContainer>
+            </StyledDeleteModal>
+        </StyledDeleteModalBackground>
+    ) : null
+    }
+    </>
+)
+
+};
+
+
+const StyledDeleteModalBackground = styled.section`
+    content: "";
```

Comment:
The `content` attribute is typically used within `::before` and `::after` pseudo elements, no need for it here.

Reply:
Yes! Thanks :)